Assets/HotmaxEditorVR/slider.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class slider : MonoBehaviour {
+
+    private bool triggerDown = false;
+    public GameObject sliderCollider;
+
+    public UnityEvent sliderUnclicked;
```

Comment:
Ok so you can make UnityEvents that have arguments passed in through the invoke, which is what we want here. 
first make a UnityEvent class with 1 float argument like so - put it on line 5 before this class: 
```
[System.Serializable]
public class SliderEvent : UnityEvent<float>
{}
```
then on line 11 write `public SliderEvent sliderUnClicked;`
then when you invoke sliderUnclicked pass in your slider's position transform z data. 

Now when your data should be passed through as an argument.

Reply:
Nice, that works great